lib/renderer/api/remote.js

```diff
@@ -259,8 +259,6 @@ function metaToPlainObject (meta) {
 // Construct an exception error from the meta.
 function metaToException (meta) {
   const error = new Error(`${meta.message}\n${meta.stack}`)
-  const remoteProcess = exports.process
```

Comment:
this leads to stack overflow as accessing process causes another exception due to remote being disabled